completion/zsh/_pacreport

```diff
@@ -0,0 +1,18 @@
+#compdef pacreport
+
+declare -a args
+args=(
+    '--backups[list .pac{save,orig,new} files]'
+    '--cachedir=[set an alternate cache location]:<path>:_path_files -/'
+    '--config=[set an alternate pacman configuration file]:<path>:_files'
+    '--dbext=[set an alternate sync database extension]:<ext>: '
+    '--dbpath=[set an alternate database location]:<path>:_files'
+    '*--group=[list missing group packages]:<group>:{_values group $(pacman -Sg)}'
+    '(-)--help[display help information]'
+    '--missing-files[list missing package files]'
+    '--root=[set an alternate installation root]:<path>:_path_files -/'
+    '--sysroot=[set an alternate system root]:<path>:_path_files -/'
+    '--unowned-files[list unowned files]'
+    '(-)--version[display version information]'
```

Comment:
While looking into bash completion for `pacreport` I noticed that it also accepts `--optional-deps`, which is undocumented but mentioned in the man pages.

Reply:
Huh, I can look into that.

I based the available options on `--help`, and only referred to the man page to figure out argument completion. Plus I noticed that `--version` and `--help` were sometimes missing from `--help` so I added those as well where missing. A separate PR should be created to fix issues in `--help` for the various commands.

Reply:
@sol I opened issue #57 to track the discrepancies.